test/e2e/operator_test.go

```diff
@@ -159,6 +160,14 @@ func TestMain(m *testing.M) {
 		fmt.Printf("Failed to create rolebinding external-dns-operator in ns %s: %v\n", operandNamespace, err)
 	}
 
+	if err = ensureOperandClusterRoleBinding(); err != nil {
+		fmt.Printf("Failed to create clusterrolebinding external-dns : %v\n", err)
+	}
+
+	if err = ensureOperandClusterRole(); err != nil {
+		fmt.Printf("Failed to create clusterrole external-dns in : %v\n", err)
```

Comment:
I think you can pass t *testing.T in the function and then add t.Errorf rather than using fmt.Printf